packages/taraxa-ui/src/components/Notification/Notification.tsx

```diff
@@ -20,30 +21,41 @@ const Notification = ({
 }: React.PropsWithChildren<NotificationProps>) => {
   const classes = useStyles();
   let containerStyle = classes.container;
+  let iconColor = '#E96828';
 
   if (variant) {
     if (variant === 'success') {
       containerStyle += ` ${classes.success}`;
+      iconColor = '#15AC5B';
     }
     if (variant === 'danger') {
       containerStyle += ` ${classes.danger}`;
+      iconColor = '#FF515A';
     }
     if (variant === 'info') {
       containerStyle += ` ${classes.info}`;
+      iconColor = '#48BDFF';
     }
   }
 
   return (
     <ThemeProvider theme={theme}>
       <CssBaseline />
       <div className={containerStyle}>
-        {title && <Text label={title} variant="body1" color="primary" className={classes.title} />}
-        {text && <Text label={text} variant="body2" color="primary" className={classes.text} />}
-        {children && (
-          <Text variant="body2" color="primary" className={classes.text}>
-            {children}
-          </Text>
-        )}
+        <div className={classes.icon}>
```

Comment:
Would be nice to have a selector props for this to be applied or not, most likely we'll need it.

Reply:
@VargaElod23 What are you referring specifically?

Reply:
I meant a toggle prop like `hasNotification: boolean` which determines if the icon is shown or not.

Reply:
I meant a toggle prop like `hasNotification: boolean` which determines if the icon is shown or not.

Reply:
Based on the issue we always need to show an icon:
![imagen](https://user-images.githubusercontent.com/602693/160684864-46c833f5-1800-42bf-a6c5-50caff2dd990.png)

Reply:
Okay. Closed 💪🏻